### ISSUE TITLE
[java] False Positive: CommentDefaultAccessModifier in inner interfaces of an interface

### ISSUE THREAD

**Issue by martofeld** [pmd/pmd#1257]
**Affects PMD Version:** 
6.5.0
**Rule:**
CommentDefaultAccessModifier
**Description:**
The following error occurs
`To avoid mistakes add a comment at the beginning of the InnerInterface nested class if you want a default access modifier`

Since the class is inner to an interface by default the visibility should be public and modifiers should not be needed

**Code Sample demonstrating the issue:**

```
public @interface MyInterface {
    void method1();

    interface InnerInterface {}
}
```

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*

**Comment by jsotuyod** [comment]
This has already been fixed in 6.6.0 when addressing #1211
